.github/workflows/main.yml

```diff
@@ -210,11 +209,10 @@ jobs:
           micromamba install -c conda-forge "pyjuliacall>=0.9.20"
       - name: Install xHydro
         run: |
-          python -m pip install --no-deps .
+          python -m pip install .
```

Comment:
Not a fan of this. The conda package should be 100%-compatible with the `pyproject.toml` dependencies.

Reply:
Alright, I've reverted this.